### PR TITLE
478.5/optimistic liquidity

### PR DESCRIPTION
**Pull request by W3stside**
Adds same as #498 but for liquidity orders

### CODE REVIEW

**Comment by W3stside** [comment]
@Velenir @alfetopito just check the last commit, same thing is happening as last time

**Comment by gnosis-info** [comment]
Travis automatic deployment:
 https://pr512--dexreact.review.gnosisdev.com

File: src/components/PoolingWidget/index.tsx [hunk 1]
```diff
@@ -207,24 +210,38 @@ const PoolingInterface: React.FC = () => {
...
 
       const { receipt } = await placeMultipleOrders({
         orders,
         txOptionalParams: {
-          onSentTransaction,
+          onSentTransaction: (txHash: string): void => {
+            pendingTxHash = txHash
+            setTxHash(txHash)
+
+            orders.map(({ buyToken: buyTokenId, sellToken: sellTokenId, buyAmount, sellAmount }) => {
```
Comment (Velenir) [inline src/components/PoolingWidget/index.tsx]: use `.forEach`

File: src/components/PoolingWidget/index.tsx [hunk 2]
```diff
@@ -207,24 +210,38 @@ const PoolingInterface: React.FC = () => {
...
+                priceNumerator: buyAmount,
+                priceDenominator: sellAmount,
+                user: userAddress,
+                remainingAmount: ZERO,
+                sellTokenBalance: ZERO,
+                validFrom: 0,
+                validUntil: 0,
+              }
+
+              return dispatch(savePendingOrdersAction({ orders: newTxState, networkId, userAddress }))
```
Comment (Velenir) [inline src/components/PoolingWidget/index.tsx]: I wonder if multiple dispatches in a row cause rerenders
If you could check and maybe try with `unstable_batchedUpdates`

**Comment by alfetopito** [comment]
> @Velenir @alfetopito just check the last commit, same thing is happening as last time

```
git rebase -i origin/develop

# edit the file and replace `pick` with `f` from the second commit until the one before what you want to keep
# save file and rebasing magic will happen

git rebase origin/develop

File: src/components/PoolingWidget/index.tsx (hunk 1 above)
Comment (W3stside) [reply src/components/PoolingWidget/index.tsx]: good catch

File: src/components/PoolingWidget/index.tsx (hunk 2 above)
Comment (W3stside) [reply src/components/PoolingWidget/index.tsx]: :+1:

**Comment by W3stside** [comment]
> > @Velenir @alfetopito just check the last commit, same thing is happening as last time
> 
> ```
> git rebase -i origin/develop
> 
> # edit the file and replace `pick` with `f` from the second commit until the one before what you want to keep
> # save file and rebasing magic will happen
> 
> git rebase origin/develop
> ```

or I force the burden onto you guys reviewing instead :grin: jk, will do this next time

**Pull request merged**